docs/source/releases/v3.0.rst

```diff
@@ -19,7 +19,7 @@ also need to upgrade their frontend and dashboard templates to use Bootstrap 4.
 Compatibility
 ~~~~~~~~~~~~~
 
-Oscar 3.0 is compatible with Django 2.2, Django 3.0 and Django 3.1 as well as Python 3.6, 3.7 and 3.8.
```

Comment:
Please revert the changes to 3.0 release notes (3.0 has already shipped). This change will ship in Oscar 3.1 and we'll update the notes for 3.1 before release (since we'll also be dropping support for Django 3.0 which is now end of life).

Reply:
Will do